homeassistant/components/getsequence/config_flow.py

```diff
@@ -0,0 +1,230 @@
+    CONF_INVESTMENT_ACCOUNTS,
+    CONF_LIABILITY_ACCOUNTS,
+    CONF_LIABILITY_CONFIGURED,
+    DOMAIN,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_ACCESS_TOKEN): str,
+    }
+)
+
+
+async def validate_input(hass: HomeAssistant, data: dict[str, Any]) -> dict[str, Any]:
+    """Validate the user input allows us to connect.
+
+    Data has the keys from STEP_USER_DATA_SCHEMA with values provided by the user.
+    """
+    session = async_get_clientsession(hass)
+    client = SequenceApiClient(session, data[CONF_ACCESS_TOKEN])
+
+    # Test the connection and get account info for unique ID
+    accounts_data = await client.async_get_accounts()
+
+    # We'll use the first pod account ID as unique identifier, or fallback to a hash of token
+    pod_accounts = client.get_pod_accounts(accounts_data)
+    if pod_accounts:
+        unique_id = str(pod_accounts[0]["id"])
+        title = f"Sequence ({len(pod_accounts)} pods)"
+    else:
+        # Fallback if no pods exist yet
+        unique_id = data[CONF_ACCESS_TOKEN][-8:]  # Last 8 chars of token
+        title = "Sequence"
+
+    return {"title": title, "unique_id": unique_id}
+
+
+class SequenceConfigFlow(ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for Sequence."""
+
+    @staticmethod
+    @callback
+    def async_get_options_flow(config_entry: ConfigEntry) -> SequenceOptionsFlow:
+        """Get the options flow for this handler."""
+        return SequenceOptionsFlow()
+
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> ConfigFlowResult:
+        """Handle the initial step."""
+        errors: dict[str, str] = {}
+
+        if user_input is not None:
+            try:
+                info = await validate_input(self.hass, user_input)
+            except SequenceAuthError:
+                errors["base"] = "invalid_auth"
+            except SequenceConnectionError:
+                errors["base"] = "cannot_connect"
+            except SequenceApiError:
+                errors["base"] = "unknown"
+            except Exception:
+                _LOGGER.exception("Unexpected exception")
+                errors["base"] = "unknown"
+            else:
+                await self.async_set_unique_id(info["unique_id"])
+                self._abort_if_unique_id_configured()
+                return self.async_create_entry(
+                    title=info["title"],
+                    data=user_input,
+                )
+
+        return self.async_show_form(
+            step_id="user",
+            data_schema=STEP_USER_DATA_SCHEMA,
+            errors=errors,
+        )
+
+    async def async_step_reauth(
+        self, entry_data: Mapping[str, Any]
+    ) -> ConfigFlowResult:
+        """Handle reauth flow."""
+        return await self.async_step_reauth_confirm()
+
+    async def async_step_reauth_confirm(
+        self, user_input: dict[str, Any] | None = None
```

Comment:
should we keep reauth out for now so we can focus on the rest first?

Reply:
We could... that said I hate deleting working code. Would it be acceptable for me to comment this out until it can be reviewed later?